src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -1,16 +1,19 @@
 <template>
-	<www-page class="tw-bg-secondary" style="height: auto;">
-		<div class="tw-max-w-5xl tw-mx-auto tw-p-2 lg:tw-pt-4">
-			<h3 class="tw-text-h3 tw-text-primary">
-				Welcome back, <span class="tw-text-action fs-mask data-hj-suppress">{{ firstName }}</span>
-			</h3>
+	<www-page class="tw-bg-white" style="height: auto;">
+		<div class="tw-w-full">
+			<!-- eslint-disable-next-line max-len -->
+			<div class="tw-mx-auto tw-p-2 lg:tw-pt-4 tw-px-2.5 md:tw-px-4 lg:tw-px-8 tw-bg-white" style="max-width: 1200px;">
```

Comment:
Can this max width just be in this component in a wrapper div (as opposed to on each component)?

Reply:
No b/c every section need to be full width to show the bg correctly and the content inside within 1200px width

Reply:
Makes sense, thanks for considering.

Reply:
Having `tw-bg-white` on line 5 here is redundant now that the page bg is white.

Reply:
right, removed